POEApi.Model/LegionType.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace POEApi.Model
+{
+    public enum LegionType
+    {
+        EternalEmpire,
+        Karui,
+        Maraketh,
+        Templar,
+        Vaal, 
+
+        Unknown
```

Comment:
It's not generally done with enums in Procurement, but we should make `Unknown` the default value (by making it [have a value of 0](https://stackoverflow.com/a/4967673)).